app/views/provider_interface/reports/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, t('page_titles.provider.reports') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <%= t('page_titles.provider.reports') %>
+    </h1>
+    <ul class="govuk-list">
+      <% if FeatureFlag.active?(:export_application_data) %>
+        <li>
+          <%= govuk_link_to 'Export application data', provider_interface_new_application_data_export_path %>
```

Comment:
Could we use a locale string here (and for the link below). The text in the links here should match the H1s on the pages too, going by the prototype